src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+import duke.exception.*;
+
+import java.io.IOException;
+
+public class Duke {
+    public static void main(String[] args) {
+        Init.showWelcomeMessage();
+        try {
+            Save.loadData();
+        } catch(IOException e) {
+            System.out.println(e.getMessage());
+        }
+
+        while(true) {
```

Comment:
Be careful with using `true` as a `while` loop condition without specifying the termination criteria elsewhere. Perhaps replacing this with a termination condition or variable would be more meaningful.